README.md

```diff
@@ -1 +1,39 @@
 # duda-migration-utils
+
+## Setup
+
+```bash
+npm install @dudadev/duda-migration-utils
+```
+
+## API
+
+## `getDomainUrl`
+
+Usage example:
+
+```js
+const { getDomainUrl } = require('@dudadev/duda-migration-utils');
+
+const domainUrl = await getDomainUrl({ zipModifier });
```

Comment:
Here as well - maybe have `zipModifier` as the first argument in all of the helper functions. (And have the options object after that).